components/Snackbar/src/MDCSnackbarManager.h

```diff
@@ -51,6 +51,11 @@
  */
 @interface MDCSnackbarManager : NSObject
 
+/**
+ An instance of MDCSnackbarManager.
+ */
+@property(class, nonnull, nonatomic, readonly, strong) MDCSnackbarManager *defaultManager;
```

Comment:
Will the prevent us from ever presenting a snackbar on a screen that isn't mainScreen?

Reply:
Nope, it would in fact permit greater flexibility in binding snackbar managers to windows.